pkg/skyenv/values_darwin.go

```diff
@@ -8,6 +8,10 @@ const (
 	OS = "mac"
 	// SkywirePath is the path to the installation folder.
 	SkywirePath = "/Library/Application Support/Skywire"
+	// Ptext is the description to use for the -p flag for config gen and for the visor
+	Ptext = "use mac installation path: " + SkywirePath
```

Comment:
I think you misunderstood my request. I would like to get rid of the additional global variable. We can just construct this message inside the flag declaration instead of exposing a global variable.

Reply:
done with [f672c82](https://github.com/skycoin/skywire/pull/1157/commits/f672c824991c377eca6180c4d63158c7ea9dd1b1)